saleor/graphql/api.py

```diff
@@ -116,7 +117,7 @@ class Mutation(
 schema = build_federated_schema(
     Query,
     mutation=Mutation,
-    types=unit_enums + list(WEBHOOK_TYPES_MAP.values()),
+    types=unit_enums + list(WEBHOOK_TYPES_MAP.values()) + list(APP_EVENTS_MAP.values()),
```

Comment:
Are you sure that you need this? 
AFAIK it should not be necessary. The app events should be imported through the `App` type.